immuni_common/helpers/sanic.py

```diff
@@ -178,3 +184,52 @@ def _validate_json_content_type(request: Request) -> None:
         return _wrapper
 
     return _decorator
+
+
+async def wait_configured_time() -> None:
+    """
+    Wait for the configured time.
+    This is usually useful to make dummy requests last a similar amount of time when compared to the
+    real ones, or slow down potential brute force attacks.
+    """
+    await asyncio.sleep(
+        random.normalvariate(
+            config.DUMMY_REQUEST_TIMEOUT_MILLIS, config.DUMMY_REQUEST_TIMEOUT_SIGMA
+        )
+        / 1000.0
+    )
+
+
+def handle_dummy_requests(
```

Comment:
IMHO, it should be `handle_dummy_request`, since it handles one request at a time. What's your take?

Reply:
I called it like that because I think it improves visibility when used as a decorator of the endpoints:

```
@route(...)
@handle_dummy_requests
def my_endpoint(...):
```

I think it's clearer that the decorator allow the endpoint to generically handle the dummy requests if it's plural

Reply:
Mmh, Okay I see your point, let's keep it as is then. 🙏